app/controllers/application_controller/ci_processing.rb

```diff
@@ -605,9 +625,16 @@ def process_resourcepools(rps, task)
   #   operation   - a block with a operation, specific to the type of action
   #                 on a record
   #   options     - other optional parameters
+  #               - :redirect => :controller
+  #                           => :action
+  #                 :refresh_partial
+  #                 used by #screen_redirection
+  #               - :specific_requirements
+  #                 used by #specific_requirements_fullfiled?
   def generic_button_operation(action, action_name, operation, options = {})
     records = find_records_with_rbac(get_rec_cls, checked_or_params)
-    if testable_action(action) && !records_support_feature?(records, action_to_feature(action))
+    if (testable_action?(action) && !records_support_feature?(records, action_to_feature(action))) ||
+       !specific_requirements_fullfiled?(records, options[:specific_requirements])
```

Comment:
the word in the above method is spelled "fulfilled" - can you change the method spelling here and in its definition in line 660?  thanks.

Reply:
@jerryk55 does the fix resolve the reported problem?